<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="Gowthami" failures="2" tests="2" name="Submit Order" time="49.556" errors="0" timestamp="2025-02-09T22:04:23 PST">
  <testcase classname="Automationacademy.tests.SubmitOrderTest" name="SubmitOrder" time="20.749">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Automationacademy.tests.SubmitOrderTest.SubmitOrder(SubmitOrderTest.java:48)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- SubmitOrder -->
  <testcase classname="Automationacademy.tests.SubmitOrderTest" name="SubmitOrder" time="20.312">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Automationacademy.tests.SubmitOrderTest.SubmitOrder(SubmitOrderTest.java:48)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- SubmitOrder -->
  <testcase classname="Automationacademy.tests.SubmitOrderTest" name="OrderHistoryTest" time="0.0">
    <ignored/>
  </testcase> <!-- OrderHistoryTest -->
</testsuite> <!-- Submit Order -->
